File: alphabeticShift/alphabeticShift.js
alphabeticShift = str =>
    str
    .split("")
    .map(i => String.fromCharCode(
        (i.charCodeAt(0) % 122 == 0) ?
        97 :
        i.charCodeAt(0) + 1))
    .join("");

console.log(alphabeticShift('crazy'));
console.log(alphabeticShift('aaavd'));

File: Completed/js/alphabeticShift/alphabeticShift.py
def alphabeticShift(s):
    s = list(s)
    for i in range(len(s)):
        s[i] = "a" if ord(s[i]) % 122 == 0 else chr(ord(s[i]) + 1)
    return "".join(s)


print(alphabeticShift("crazy"))
